airbyte-integrations/connectors/source-plaid/source_plaid/manifest.yaml

```diff
@@ -145,14 +145,38 @@ streams:
             type:
               - string
               - "null"
+          counterparties:
```

Comment:
Leaving a comment here, it looks like `category` and `category_id` are deprecated in `transactions/sync` do we also want to remove those?

Reply:
I went back and forth about this—our code currently reads them, so I settled on continuing to import them for now. But I'll file a ticket to remove them once we fix our bank sync code.